### PR TITLE
Make filter labels consistent across desctop/mobile

### PR DESCRIPTION
**Pull request by 74-921**

### CODE REVIEW

File: frontend/app/views/spree/products/_filters_desktop.html.erb [hunk 1]
```diff
@@ -47,7 +47,7 @@
             <% price_param = params[:price] == name ? '' : name %>
             <%= link_to permitted_params.merge({ price: price_param }), class: "d-block" do %>
               <div class="d-inline-block py-1 px-2 m-1 text-uppercase plp-filters-card-item <%= 'plp-filters-card-item--selected' if params[:price] == name %>">
-                <%= name %>
+                <%= name.upcase %>
```
Comment (damianlegawiec) [inline frontend/app/views/spree/products/_filters_desktop.html.erb]:
```suggestion
                <%= name%>
```

@Punkbooster we don't need to do this, there's already `text-uppercase` CSS class attached

File: frontend/app/views/spree/products/_filters_mobile.html.erb [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
         <div class="card plp-overlay-card">
           <div class="card-header text-uppercase <%= 'collapsed' if params[:price].blank? %> plp-overlay-card-header" id="headingFour" data-toggle="collapse" data-target="#collapseFour" aria-expanded="true" aria-controls="collapseFour" role="heading" aria-level="3">
-            <%= Spree.t('plp.price') %>
+            <%= Spree.t('plp.price').upcase %>
```
Comment (damianlegawiec) [inline frontend/app/views/spree/products/_filters_mobile.html.erb]: @Punkbooster same here

File: frontend/app/views/spree/shared/_option_values.html.erb [hunk 3]
```diff
@@ -16,7 +16,7 @@
                       </span>
       <% else %>
         <div class="d-inline-block py-1 px-2 m-1 plp-overlay-card-item <%= 'plp-overlay-card-item--selected' if selected_option_values.include?(id) %>">
```
Comment (damianlegawiec) [inline frontend/app/views/spree/shared/_option_values.html.erb]:
```suggestion
        <div class="text-uppercase d-inline-block py-1 px-2 m-1 plp-overlay-card-item <%= 'plp-overlay-card-item--selected' if selected_option_values.include?(id) %>">
```

File: frontend/app/views/spree/shared/_option_values.html.erb [hunk 4]
```diff
@@ -16,7 +16,7 @@
...
-          <%= name %>
+          <%= name.upcase %>
```
Comment (damianlegawiec) [inline frontend/app/views/spree/shared/_option_values.html.erb]:
```suggestion
          <%= name %>
```

File: frontend/app/views/spree/products/_filters_desktop.html.erb [hunk 5]
```diff
@@ -31,7 +31,7 @@
 
     <div class="w-100 card plp-filters-card">
       <div class="card-header <%= 'collapsed' if params[:price].blank? %> px-1 plp-filters-card-header" id="filtersPrice" data-toggle="collapse" data-target="#collapseFilterPrice" aria-expanded="false" aria-controls="collapseFilterPrice" role="heading" aria-level="2">
-        <%= Spree.t('plp.price') %>
+        <%= Spree.t('plp.price').capitalize %>
```
Comment (damianlegawiec) [inline frontend/app/views/spree/products/_filters_desktop.html.erb]:
```suggestion
        <%= Spree.t('plp.price') %>
```

File: frontend/app/views/spree/products/_filters_desktop.html.erb [hunk 6]
```diff
@@ -31,7 +31,7 @@
 
     <div class="w-100 card plp-filters-card">
       <div class="card-header <%= 'collapsed' if params[:price].blank? %> px-1 plp-filters-card-header" id="filtersPrice" data-toggle="collapse" data-target="#collapseFilterPrice" aria-expanded="false" aria-controls="collapseFilterPrice" role="heading" aria-level="2">
```
Comment (damianlegawiec) [inline frontend/app/views/spree/products/_filters_desktop.html.erb]:
```suggestion
      <div class="card-header <%= 'collapsed' if params[:price].blank? %> px-1 plp-filters-card-header text-capitalize" id="filtersPrice" data-toggle="collapse" data-target="#collapseFilterPrice" aria-expanded="false" aria-controls="collapseFilterPrice" role="heading" aria-level="2">
```

**Review by damianlegawiec — APPROVED** [review]

**Review by pmilewski — APPROVED** [review]

**Pull request merged**